File: 프로그래머스/unrated/155652. 둘만의 암호/둘만의 암호.js
function solution(s, skip, index) {
    let alphabet = "abcdefghijklmnopqrstuvwxyz";
    
    for (let i = 0; i < skip.length; i++) {
        alphabet = alphabet.replace(skip[i], "");
    }

    let result = "";
    for (let i = 0; i < s.length; i++) {
        let idx = alphabet.indexOf(s[i])+index;
        
        while (alphabet.length <= idx) {
        idx = idx - alphabet.length;
        }
        
        result += alphabet[idx];
    }
    return result;
    }

File: 프로그래머스/1/155652. 둘만의 암호/둘만의 암호.py
def solution(s, skip, index):
    alpha = [chr(x) for x in range(97, 123)]
    
    for i in skip:
        alpha.remove(i) # 스킵할 문자 삭제

    result = ''
    for char in s:
        result += alpha[(alpha.index(char) + index) % len(alpha)]
    return result
    
    
    
        